src/styles/index.scss

```diff
@@ -1,3 +1,75 @@
+@import 'variables';
+
 body {
   margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  position: relative;
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  &_minutes-hand {
+    position: absolute;
+    bottom: 50%;
+    left: 50%;
+    width: $minutes-width;
+    height: $minutes-height;
+    transform: $hands-transform;
+    background-color: #0700ff;
+    z-index: 1;
+    transform-origin: 50% 100%;
+    animation: rotate 3600s infinite steps(60);
+  }
+
+  &_seconds-hand {
+    position: absolute;
+    bottom: 50%;
+    left: 50%;
+    width: $seconds-width;
+    height: $seconds-height;
+    background-color: #2c8000;
+    z-index: 2;
+    transform: $hands-transform;
+    transform-origin: 50% 100%;
+    animation: rotate 60s infinite linear;
+  }
+
+  &_center-circle {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    width: 5vmin;
+    height: 5vmin;
+    background-color: #f6a603;
+    border-radius: 50%;
+    z-index: 3;
+  }
+}
+
+.stopwatch--speed-up {
+  .stopwatch_minutes_hand {
+    animation: rotate 600s infinite steps(60);
+  }
+
+  .stopwatch_seconds_hand {
+    animation: rotate 10s infinite linear;
+  }
+}
+
+@keyframes rotate {
+  from {
+    transform: translate(-50%, 0%) rotate(0deg);
+  }
+  to {
+    transform: translate(-50%, 0%) rotate(360deg);
+  }
 }
```

Comment:
The `rotate` animation is used multiple times in your code. To avoid repetition and make your code DRY (Don't Repeat Yourself), you could consider creating a mixin for this animation and include it where needed.